app/code/Magento/InventoryInStorePickup/Test/Integration/Extension/InventorySourceExtensionTest.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryInStorePickup\Test\Integration\Extension;
+
+use Magento\Framework\ObjectManagerInterface;
+use Magento\InventoryApi\Api\SourceRepositoryInterface;
+use Magento\TestFramework\Helper\Bootstrap;
+use PHPUnit\Framework\TestCase;
+
+class InventorySourceExtensionTest extends TestCase
+{
+    /** @var ObjectManagerInterface */
+    private $objectManager;
+
+    /** @var SourceRepositoryInterface */
+    private $sourceRepository;
+
+    protected function setUp()
+    {
+        $this->objectManager = Bootstrap::getObjectManager();
+
+        $this->sourceRepository = $this->objectManager->get(SourceRepositoryInterface::class);
+    }
+
+    /**
+     * @magentoDataFixture ../../../../app/code/Magento/InventoryApi/Test/_files/sources.php
+     */
+    public function testSaveSourcesWithActivePickupLocation()
```

Comment:
Test names are slightly misleading, I couldn't figure out at first that it tests save and load at the same time.

Reply:
Renamed.
Please re-check.